src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo__link">
+        <img
+        src="images/logo.png"
+        alt="MOYO logo"
+        class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__items">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__items">
+            <a href="#smarphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__items">
+            <a
+            href="#laptops&computers"
+            class="nav__link"
+            data-qa="hover"
+            >
```

Comment:
Please make indentation here

```suggestion
            <a
              href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
```